### PR TITLE
Add clientside tests for Facilitator Program Registration

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: apps/test/unit/facilitatorProgramRegistration.js [hunk 1]
```diff
@@ -0,0 +1,235 @@
+import {assert} from '../util/configuredChai';
```
Comment (mehalshah) [inline apps/test/unit/facilitatorProgramRegistration.js]: I think convention is to have the word test in the file name

File: apps/test/unit/facilitatorProgramRegistration.js [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+testUtils.setExternalGlobals();
+import React from 'react';
+import {mount} from 'enzyme';
+
+import FacilitatorProgramRegistration from '@cdo/apps/code-studio/pd/facilitator_program_registration/FacilitatorProgramRegistration';
+
+const DEFAULTS = {
+  apiEndpoint: "",
+  course: "",
+  options: {
```
Comment (mehalshah) [inline apps/test/unit/facilitatorProgramRegistration.js]: Are these options we can just import from the source js?

File: apps/test/unit/facilitatorProgramRegistration.js [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+          },
+          training: {
+            arrive: "",
+            depart: ""
+          }
+        }}
+      />
+    );
+
+    // by default, "next" and not "submit"
```
Comment (mehalshah) [inline apps/test/unit/facilitatorProgramRegistration.js]: You can probably DRY this section. Your call if you want to do that

File: apps/test/unit/facilitatorProgramRegistration.js (hunk 2 above)
Comment (Hamms) [reply apps/test/unit/facilitatorProgramRegistration.js]: nope! Options are by design actually supplied by the server (from `Pd::FacilitatorProgramRegistration::OPTIONS`)

**Pull request merged**